Client/core/CCommandFuncs.cpp

```diff
@@ -21,12 +21,6 @@ extern bool   g_bBoundsChecker;
 void CCommandFuncs::Help(const char* szParameters)
 {
     CConsoleInterface* pConsole = g_pCore->GetConsole();
-
-    if (szParameters && atoi(szParameters) == 1)
-        CCore::GetSingleton().GetMultiplayer()->AllowWindowsCursorShowing(true);
-    else
-        CCore::GetSingleton().GetMultiplayer()->AllowWindowsCursorShowing(false);
-
```

Comment:
These changes come from initial commit, seems reasonable to remove 👍

Reply:
Merged in 0cdc4d2d53dd7f829767e1f790ea47b29fc1ff31.